download_sdl2_hg.cmake

```diff
@@ -26,11 +27,12 @@ endif()
 ExternalProject_Add(
     SDL2HG
     PREFIX ${CMAKE_BINARY_DIR}/external/SDL2
-    URL https://hg.libsdl.org/SDL/archive/default.tar.bz2
+    GIT_REPOSITORY https://github.com/spurious/SDL-mirror.git
     CMAKE_ARGS
         "-DCMAKE_BUILD_TYPE=${CMAKE_BUILD_TYPE}"
         "-DCMAKE_INSTALL_PREFIX=${CMAKE_BINARY_DIR}"
         -DSNDIO=OFF
+        -DSDL_SHARED=${SHARED}
```

Comment:
Let's this will be a part of this?
https://github.com/WohlSoft/AudioCodecs/issues/5

Reply:
Anyway, as it's undefined now, it will cause an error... Please define next options:
```cmake
option(AudioCodecs_SDL2_STATIC ...)
option(AudioCodecs_SDL2_SHARED ...)
```
which are both will be ON by default.
Then, pass them as:
```
-DSDL_SHARED=${AudioCodecs_SDL2_SHARED}
-DSDL_STATIC=${AudioCodecs_SDL2_STATIC}
```